### DISCUSSION TITLE
Spend management: enable "pause project and stop usage" configuration via GUI, not just via webhook

### DISCUSSION THREAD

**Discussion by peterhartree** [vercel/community#5941] [Idea]
I wish I could just switch this on from the settings UI, rather than having to go through the faff of handling a webhook.

Once implemented, this should probably be "on" by default for new projects (with suitable flagging during the project creation process).

Screenshot from [the docs](https://vercel.com/docs/accounts/spend-management):
<img width="864" alt="image" src="https://github.com/vercel/community/assets/1412568/1dfa9540-284d-4ae4-a057-6c42d2b95202">

The recent [Netlify debacle](https://news.ycombinator.com/item?id=39520776) brought this issue to mind.

**Comment by leerob** [comment]
Thank you for this suggestion! This is a good idea and we've heard similar feedback.
